README.md

```diff
@@ -1191,9 +1191,22 @@ GET /apps/:guid?fields=expensive_field
 }
 ```
 
-### Proposal: Fields For Sub-Resources
+### Fields For Sub-Resources
+
+This is a mechanism for including multiple fields of related resources in a single response.
+
+Resources and collections **may** accept a `fields[path]` query parameter for each related resources with a list of resource fields. 
+
+Each resource path **MUST** be a series of period-separated relationship names. For example: `app.space.organization`. 
+
+The list of resource fields **MUST** be comma delimited. For example: `fields[space]=name&fields[space.organization]=name,guid`.
+
+Included resource fields **MUST** be returned in an `included` object on the primary resource or collection.
```

Comment:
No, that is something that we have been doing in some of the resources for convenience. Can update the example to leave them out so that it is clearer.

Reply:
Is there an advantage to including the name in the relationship vs using the `includes` parameter? Also, if we want to do this more generically, not all resources have `name` fields. Maybe we could use a more generic field like `presentation_name` or something.

Reply:
I updated the example to not include the name since it is not something that is done for all resources (we did for some service resources since it's more useful than guide). This PR is not intended to add "name" into the "relationships" part of the resource. Maybe that's something we should discuss separately.